.gitignore

```diff
@@ -8,3 +8,13 @@
 /var/
 /vendor/
 ###< symfony/framework-bundle ###
+
+###> phpunit/phpunit ###
+/phpunit.xml
+.phpunit.result.cache
+###< phpunit/phpunit ###
+
+###> symfony/phpunit-bridge ###
+.phpunit.result.cache
+/phpunit.xml
+###< symfony/phpunit-bridge ###
```

Comment:
Duplicate recipes for `phpunit/phpunit` && `symfony/phpunit-bridge`

Reply:
`composer require symfony/test-pack` adds `phpunit/phpunit` && `symfony/phpunit-bridge` to the application among other dependencies. 

Flex installs the `phpunit-bridge` recipe https://github.com/symfony/recipes/blob/master/symfony/phpunit-bridge/5.3/manifest.json && the `phpunit` recipe https://github.com/symfony/recipes/blob/master/phpunit/phpunit/9.3/manifest.json

Both have duplicate `.gitignore` entries, but because there are "prefixed" by different packages in the comments, the duplicate is allowed?